vtex/toolbelt
# Make deprecation checks non-blocking

#### What is the purpose of this pull request?
Inspired by `update-notifier` code, this makes deprecations checks non-blocking by creating a process to ask NPM if the package is deprecated

#### How should this be manually tested?
```
cd /tmp && rm -rf toolbelt && \
git clone https://github.com/vtex/toolbelt.git && \
cd toolbelt && \
git checkout fix/cli-prechecks && \
yarn && yarn watch
```
You can go to `package.json` and change the version to a deprecated version, `2.86.0` for example 
The file created to keep track of the last deprecation check and last check result is on `~/.config/confgstore`

#### Types of changes

#### Chores checklist
